src/controller.js

```diff
@@ -0,0 +1,88 @@
+// controller.js
+
+export default class Controller {
```

Comment:
one controller for all actions and views? weird

additionally `requestData` is part of the data